code/modules/reagents/chemistry/equilibrium.dm

```diff
@@ -44,6 +44,8 @@
 	var/thermic_mod = 1
 	///Allow us to deal with lag by "charging" up our reactions to react faster over a period - this means that the reaction doesn't suddenly mass react - which can cause explosions
 	var/time_deficit
+	///Used to store specific data needed for a reaction
+	var/data = list()
```

Comment:
Could you give this a more informative name?
What kind of data?

Reply:
All kinds of data! But I'll update the comment

Reply:
Hopefully that gets across what it's for